src/S3/S3Client.php

```diff
@@ -219,6 +219,8 @@ public static function isBucketDnsCompatible($bucket)
 
     public function createPresignedRequest(CommandInterface $command, $expires)
     {
+        $command->getHandlerList()->remove('signer');
```

Comment:
I looked into this, and commands are used up when executed or presigned. I can push up a change that clones the command, but I wasn't sure if it was in scope for this PR.